src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,532 @@
+  const { auth } = useChrome();
+
+  useEffect(() => {
+    (async () => {
+      const userData = await auth?.getUser();
+      const id = userData?.identity?.internal?.org_id;
+      setOrgId(id);
+    })();
+  });
+  const { isError } = useShowActivationKeyQuery(
+    // @ts-ignore - type of 'activationKey' might not be strictly compatible with the expected type for 'name'.
+    { name: activationKey },
+    {
+      skip: !activationKey,
+    }
+  );
+  return (
+    <>
+      <TextContent>
+        <TextList component={TextListVariants.dl}>
+          <TextListItem
+            component={TextListItemVariants.dt}
+            className="pf-u-min-width"
+          >
+            Registration type
+          </TextListItem>
+          <TextListItem
+            component={TextListItemVariants.dd}
+            data-testid="review-registration"
+          >
+            <TextList isPlain>
+              {registrationType?.startsWith('register-now') && (
+                <TextListItem>
+                  Register with Red Hat Subscription Manager (RHSM)
+                  <br />
+                </TextListItem>
+              )}
+              {registrationType === 'register-now-insights' ||
+                (registrationType === 'register-now-rhc' && (
+                  <TextListItem>
+                    Connect to Red Hat Insights
+                    <br />
+                  </TextListItem>
+                ))}
+              {registrationType === 'register-now-rhc' && (
+                <TextListItem>
+                  Use remote host configuration (rhc) utility
+                  <br />
+                </TextListItem>
+              )}
+            </TextList>
+          </TextListItem>
+          <TextListItem component={TextListItemVariants.dt}>
+            Activation key
+            <Popover
+              bodyContent={
+                <TextContent>
+                  <Text>
+                    Activation keys enable you to register a system with
+                    appropriate subscriptions, system purpose, and repositories
+                    attached.
+                    <br />
+                    <br />
+                    If using an activation key with command line registration,
+                    you must provide your organization&apos;s ID. Your
+                    organization&apos;s ID is{' '}
+                    {orgId !== undefined ? orgId : <Spinner size="md" />}
+                  </Text>
+                </TextContent>
+              }
+            >
+              <Button
+                variant="plain"
+                aria-label="About activation key"
+                className="pf-u-pl-sm pf-u-pt-0 pf-u-pb-0"
+                size="sm"
+              >
+                <HelpIcon />
+              </Button>
+            </Popover>
+          </TextListItem>
+          <TextListItem component={TextListItemVariants.dd}>
+            <ActivationKeyInformation />
```

Comment:
The activation key on the Review step still shows the "Selected activation key" heading. Similar to the ReleaseLifecycle we might want to solve this one in a separate PR.

Thinking about this I can see two approaches to this:

First approach is we create a description list with the needed values directly in this file. That would mean copying a long blob of `<TextContent>` code over from `ActivationKeyInformation` which would mean a lot of duplicity. Not ideal, would get rid of the `FormGroup` label.

Second option would be that we create a separate PR and do the decisions directly in the `ActivationKeyInformation` component (I would much prefer this solution). We can figure out on which step we currently are in the Wizard: https://www.patternfly.org/components/wizard#get-current-step and based on this information we can render / not render the `FormGroup`. So it would go roughly like this:
- in the ActivationKeyInformation
- get currentStep of the Wizard
- if currentStep === 'registration' render FormGroup else render just TextContent

Reply:
I agree, should be at a seperate pr

Reply:
opened this bug for that -  https://issues.redhat.com/browse/HMS-2781